.eslintrc.json

```diff
@@ -1,16 +1,19 @@
 {
 	"root": true,
-	"extends": "@aduth/eslint-config/es5",
+	"extends": "@aduth/eslint-config",
 	"parserOptions": {
-		"ecmaVersion": 5,
-		"sourceType": "script"
+		"ecmaVersion": 6,
+		"sourceType": "module"
 	},
 	"env": {
 		"node": true
 	},
 	"rules": {
+		"no-var": "off",
 		"one-var": "off",
+		"prefer-spread": "off",
 		"jsdoc/check-tag-names": "off",
-		"valid-jsdoc": "off"
+		"valid-jsdoc": "off",
+		"no-redeclare": "off"
```

Comment:
Do you want to disable these, or update the code to fix the errors?

Reply:
Was the issue because it's using the non-TypeScript version of the rule? I thought I'd seen that in an earlier commit. If it's a case of the TypeScript version working better, that sounds fine to enable the TypeScript version if the code passes with it. (The common ESLint configuration should probably be doing that by default)

Reply:
I disabled the rules to make the existing code pass. Should I change to `let` and `const` over `var` and use a spread of arguments instead of `apply`?